### PR TITLE
WIP: Review

### PR DESCRIPTION
**Pull request by majst01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ spec:
 
 ### Definition
 
-`CSI_LVM_DEVICE_PATTERN` is a grok pattern to specify which block devices to use for lvm devices on the node. This can be for example `/dev/sd[bcde]` if you want to use only /dev/sdb - /dev/sde.
+`CSI_LVM_DEVICE_PATTERN` is a grok pattern to specify which block devices to use for lvm devices on the node. This can be for example `/dev/sd[bcde]` if you want to use only /dev/sdb - /dev/sde. IMPORTANT: no wildcard (*) allowed actually.
```
Comment (droid42) [inline README.md]: i think "actually" should be "currently"

File: cmd/controller/main.go [hunk 2]
```diff
@@ -132,7 +132,7 @@ func startDaemon(c *cli.Context) error {
 		return fmt.Errorf("invalid empty flag %v", flagDefaultLVMType)
 	}
 
-	provisioner := NewLVMProvisioner(kubeClient, namespace, "/tmp/csi-lvm", devicePattern, provisionerImage, defaultLVMType)
+	provisioner := NewLVMProvisioner(kubeClient, namespace, "/data", devicePattern, provisionerImage, defaultLVMType)
```
Comment (droid42) [inline cmd/controller/main.go]: i would suggest to make "/data" a constant and add a comment, that the in-container-path of the mount must be aligned with that

File: README.md [hunk 3]
```diff
@@ -176,7 +176,7 @@ spec:
...
 
 ### PVC Striped, Mirrored
```
Comment (droid42) [inline README.md:182]: Please add a section "Local Development" or file DEVELOPMENT.md with all that loop-device magic

File: example/pod-with-linear.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    ports:
+    - containerPort: 80
+    resources:
+      limits:
+        cpu: 100m
+        memory: 100M
+  volumes:
+  - name: linear
+    persistentVolumeClaim:
+      claimName: lvm-pvc-linear
```
Comment (droid42) [inline example/pod-with-linear.yaml:24]: newline missing at end in some files

**Review by droid42 — COMMENTED** [review]
Setup instructions for local development setup would be highly appreciated :-)
Regarding the current experimental status we should add known issues and limitations to the README.

File: README.md (hunk 1 above)
Comment (majst01) [reply README.md]: done

File: cmd/controller/main.go (hunk 2 above)
Comment (majst01) [reply cmd/controller/main.go]: done

**Pull request merged**